doc/nrf/known_issues.rst

```diff
@@ -853,6 +860,14 @@ Matter (Project CHIP)
 
 .. rst-class:: v2-0-2
 
+KRKNWK-15088: Android CHIP Tool freezes on changing the sensor type
+  When you change the current sensor type after activating the monitoring of another sensor type, the application freezes.
+
+  **Workaround:** Stop the application and restart it.
+  Then, select the desired sensor type again.
```

Comment:
```suggestion
KRKNWK-15088: Android CHIP Tool shuts down on changing the sensor type
  When you change the current sensor type after activating the monitoring of another sensor type, the application shuts down.

  **Workaround:** Restart the application and select the desired sensor type again.
```

Reply:
Is this valid in 2.1.0? @Damian-Nordic reported it is not anymore.